docs/detections/api/rules/rules-api-create-single-rule-exception-item.asciidoc

```diff
@@ -0,0 +1,86 @@
+[[exceptions-api-create-rule-default-exception-item]]
+=== Create exceptions for a rule 
+
+Allows you to create exception items that are associated with a specified rule `id`.
+
+==== Request URL
+
+`POST <kibana host>:<port>/api/detection_engine/rules/<id>/exceptions`
+
+//include a tip on how to find a rule's ID
+
+==== Request body
+
+A JSON object with an array of exception items, where each exception item has the <<exceptions-api-create-exception-item,required fields>>. 
+
+[width="100%",options="header"]
+|==============================================
+|Name |Type |Description |Required
+
+|`items` |String | Specify an array of exception list items to create. |Yes 
+
+|==============================================
+
+===== Example requests
+
+[source,console]
+--------------------------------------------------
+POST api/detection_engine/rules/<id>/exceptions
```

Comment:
@yctercero would you mind providing an example request here - thank you!

Reply:
```
POST api/detection_engine/rules/<id>/exceptions
{
    items: [
        {
          "description": "Excludes the weekly maintenance job",
          "entries": [
            {
              "field": "process.name",
              "operator": "included",
              "type": "match",
              "value": "maintenance-job"
            }
          ],
          "name": "Linux maintenance job",
          "tags": [
            "in-house processes",
            "linux"
          ],
          "type": "simple"
        }
    ]
}
```

Reply:
Added in [585ed2c](https://github.com/elastic/security-docs/pull/4021/commits/585ed2cb75a83d00203852718b0a7d5fb610f4f6).